Infrastructure/Services/NotificationsClientService.cs

```diff
@@ -32,21 +37,30 @@ public class NotificationsClientService : INotificationsClientService
         _hostEnvironment = hostEnvironment;
     }
 
-    public async Task<bool> SendEmailAsync(NotificationsRecipientDto notificationsRecipient, EmailTemplateType emailTemplateType,
-        bool includeChangedFromEmailAddress = true)
+    public async Task<bool> SendEmailAsync(NotifyEmailDto notifyEmail, bool includeChangedFromEmailAddress = true)
     {
-        var clientReference = notificationsRecipient.ClientReference;
+        var clientReference = notifyEmail.ClientReference;
 
-        var emailTemplateId = GetEmailTemplateId(emailTemplateType, _notifyConfig);
+        var emailTemplateId = GetEmailTemplateId(notifyEmail.EmailTemplateType, _notifyConfig);
 
-        AddTestingInformation(notificationsRecipient, includeChangedFromEmailAddress);
+        AddTestingInformation(notifyEmail, includeChangedFromEmailAddress);
 
         try
         {
             _logger.LogInformation("Preparing to send, Notify client ref: {clientReference}", clientReference);
 
-            var result = await _notificationClient.SendEmailAsync(notificationsRecipient.Email,
-                emailTemplateId, personalisation: notificationsRecipient.Personalisation, clientReference);
+            var result = await _notificationClient.SendEmailAsync(notifyEmail.Email,
+                emailTemplateId, personalisation: notifyEmail.Personalisation, clientReference);
+
+            notifyEmail.NotifyResponse.NotifyId = result.id;
```

Comment:
Do we need this?

Reply:
We store the full Notify response details in the new `EmailNotifyResponseLog` table - so think that this needs to be passed back to populate that.  Unless I've misunderstood your question?